### PR TITLE
Fix: back icon fill

### PR DESCRIPTION
**Pull request by asaadam**
### Summary by CodeRabbit

- Style: Updated the fill color of the 'Back' arrow icon for improved visibility and consistency with the overall theme. This change enhances user interface aesthetics and improves navigation experience.

### CODE REVIEW

File: src/assets/arrow/Ic_arrow_back_circle.js [hunk 1]
```diff
@@ -3,19 +3,13 @@ import Svg, {Rect, Path} from 'react-native-svg';
...
       <Rect width={32} height={32} rx={16} fill="#000" fillOpacity={0.6} />
       <Path
-        fillRule="evenodd"
-        clipRule="evenodd"
-        d="M13.124 20.726a.888.888 0 01-1.273.011L7.27 16.154a.925.925 0 010-1.308l4.582-4.583a.888.888 0 011.273.011.929.929 0 01-.011 1.297L10.1 14.583H24.1c.497 0 .9.41.9.917a.908.908 0 01-.9.917h-14l3.013 3.012c.354.355.36.936.01 1.297z"
-        fill="#fff"
+        fill-rule="evenodd"
+        clip-rule="evenodd"
+        d="M13.1239 21.2256C12.7756 21.5868 12.2058 21.5921 11.8512 21.2373L7.26936 16.654C7.09706 16.4816 7 16.246 7 16C7 15.754 7.09706 15.5184 7.26936 15.346L11.8512 10.7627C12.2058 10.4079 12.7756 10.4132 13.1239 10.7744C13.4722 11.1355 13.4671 11.7159 13.1125 12.0707L10.1008 15.0833L24.1 15.0833C24.5971 15.0833 25 15.4937 25 16C25 16.5063 24.5971 16.9167 24.1 16.9167L10.1008 16.9167L13.1125 19.9293C13.4671 20.2841 13.4722 20.8645 13.1239 21.2256Z"
+        fill="white"
```
Comment (agit26milan) [inline src/assets/arrow/Ic_arrow_back_circle.js:12]: Ini pakai warna dari util/theme aja mas

**Review by agit26milan — CHANGES_REQUESTED** [review]

**Comment by bastian-purrer** [comment]
Image description   CodeRabbit



Currently reviewing new changes in this PR...

<details>
<summary>Commits</summary>
Files that changed from the base of the PR and between f529e187755df74abacb63d2e650152aacf75a7a and 5ea056e96fe1407b938a54d4d5142f261fd92b43 commits.
</details>

<details>
<summary>Files selected (1)</summary>

* src/assets/arrow/Ic_arrow_back_circle.js (1)
</details>






---

**Review by bastian-purrer — COMMENTED** [review]
Image description   CodeRabbit

<details>
<summary>Commits</summary>
Files that changed from the base of the PR and between f529e187755df74abacb63d2e650152aacf75a7a and 5ea056e96fe1407b938a54d4d5142f261fd92b43 commits.
</details>

<details>
<summary>Files selected (1)</summary>

* src/assets/arrow/Ic_arrow_back_circle.js (1)
</details>







<details>
<summary>Files skipped from review due to trivial changes (1)</summary>

* src/assets/arrow/Ic_arrow_back_circle.js

</details>

<details>
<summary>Review comments generated (0)</summary>

* Review: 0
* LGTM: 0

</details>

---

<details>
<summary>Tips</summary>

### Chat with Image description  CodeRabbit Bot (`@coderabbitai`)
- Reply on review comments left by this bot to ask follow-up questions. A review comment is a comment on a diff or a file.
- Invite the bot into a review comment chain by tagging `@coderabbitai` in a reply.

### Code suggestions
- The bot may make code suggestions, but please review them carefully before committing since the line number ranges may be misaligned. 
- You can edit the comment made by the bot and manually tweak the suggestion if it is slightly off.

### Pausing incremental reviews
- Add `@coderabbitai: ignore` anywhere in the PR description to pause further reviews from the bot.

</details>

**Review by bastian-purrer — COMMENTED** [review]
Image description   CodeRabbit

<details>
<summary>Commits</summary>
Files that changed from the base of the PR and between f529e187755df74abacb63d2e650152aacf75a7a and 5ea056e96fe1407b938a54d4d5142f261fd92b43 commits.
</details>

<details>
<summary>Files selected (1)</summary>

* src/assets/arrow/Ic_arrow_back_circle.js (1)
</details>







<details>
<summary>Files skipped from review due to trivial changes (1)</summary>

* src/assets/arrow/Ic_arrow_back_circle.js

</details>

<details>
<summary>Review comments generated (0)</summary>

* Review: 0
* LGTM: 0

</details>

---

<details>
<summary>Tips</summary>

### Chat with Image description  CodeRabbit Bot (`@coderabbitai`)
- Reply on review comments left by this bot to ask follow-up questions. A review comment is a comment on a diff or a file.
- Invite the bot into a review comment chain by tagging `@coderabbitai` in a reply.

### Code suggestions
- The bot may make code suggestions, but please review them carefully before committing since the line number ranges may be misaligned. 
- You can edit the comment made by the bot and manually tweak the suggestion if it is slightly off.

### Pausing incremental reviews
- Add `@coderabbitai: ignore` anywhere in the PR description to pause further reviews from the bot.

</details>

**Review by bastian-purrer — COMMENTED** [review]
Image description   CodeRabbit

<details>
<summary>Commits</summary>
Files that changed from the base of the PR and between f529e187755df74abacb63d2e650152aacf75a7a and 5ea056e96fe1407b938a54d4d5142f261fd92b43 commits.
</details>

<details>
<summary>Files selected (1)</summary>

* src/assets/arrow/Ic_arrow_back_circle.js (1)
</details>







<details>
<summary>Files skipped from review due to trivial changes (1)</summary>

* src/assets/arrow/Ic_arrow_back_circle.js

</details>

<details>
<summary>Review comments generated (0)</summary>

* Review: 0
* LGTM: 0

</details>

---

<details>
<summary>Tips</summary>

### Chat with Image description  CodeRabbit Bot (`@coderabbitai`)
- Reply on review comments left by this bot to ask follow-up questions. A review comment is a comment on a diff or a file.
- Invite the bot into a review comment chain by tagging `@coderabbitai` in a reply.

### Code suggestions
- The bot may make code suggestions, but please review them carefully before committing since the line number ranges may be misaligned. 
- You can edit the comment made by the bot and manually tweak the suggestion if it is slightly off.

### Pausing incremental reviews
- Add `@coderabbitai: ignore` anywhere in the PR description to pause further reviews from the bot.

</details>

**Review by yusufputra — APPROVED** [review]
gassss

File: src/assets/arrow/Ic_arrow_back_circle.js [hunk 2]
```diff
@@ -1,21 +1,16 @@
 import * as React from 'react';
 import Svg, {Rect, Path} from 'react-native-svg';
+import {colors} from '../../utils/colors';
```
Comment (fajarism) [inline src/assets/arrow/Ic_arrow_back_circle.js:3]: Please use src/utils/theme for color

**Review by fajarism — CHANGES_REQUESTED** [review]

**Comment by bastian-purrer** [comment]
Image description   CodeRabbit



Currently reviewing new changes in this PR...

<details>
<summary>Commits</summary>
Files that changed from the base of the PR and between f529e187755df74abacb63d2e650152aacf75a7a and 737493c3a303ae4f590517cd596caeaa8af60d8c commits.
</details>

<details>
<summary>Files selected (1)</summary>

* src/assets/arrow/Ic_arrow_back_circle.js (1)
</details>






---

**Review by bastian-purrer — COMMENTED** [review]
Image description   CodeRabbit

<details>
<summary>Commits</summary>
Files that changed from the base of the PR and between f529e187755df74abacb63d2e650152aacf75a7a and 737493c3a303ae4f590517cd596caeaa8af60d8c commits.
</details>

<details>
<summary>Files selected (1)</summary>

* src/assets/arrow/Ic_arrow_back_circle.js (1)
</details>







<details>
<summary>Files skipped from review due to trivial changes (1)</summary>

* src/assets/arrow/Ic_arrow_back_circle.js

</details>

<details>
<summary>Review comments generated (0)</summary>

* Review: 0
* LGTM: 0

</details>

---

<details>
<summary>Tips</summary>

### Chat with Image description  CodeRabbit Bot (`@coderabbitai`)
- Reply on review comments left by this bot to ask follow-up questions. A review comment is a comment on a diff or a file.
- Invite the bot into a review comment chain by tagging `@coderabbitai` in a reply.

### Code suggestions
- The bot may make code suggestions, but please review them carefully before committing since the line number ranges may be misaligned. 
- You can edit the comment made by the bot and manually tweak the suggestion if it is slightly off.

### Pausing incremental reviews
- Add `@coderabbitai: ignore` anywhere in the PR description to pause further reviews from the bot.

</details>

**Review by fajarism — APPROVED** [review]

**Review by bastian-purrer — COMMENTED** [review]
Image description   CodeRabbit

<details>
<summary>Commits</summary>
Files that changed from the base of the PR and between f529e187755df74abacb63d2e650152aacf75a7a and 737493c3a303ae4f590517cd596caeaa8af60d8c commits.
</details>

<details>
<summary>Files selected (1)</summary>

* src/assets/arrow/Ic_arrow_back_circle.js (1)
</details>







<details>
<summary>Files skipped from review due to trivial changes (1)</summary>

* src/assets/arrow/Ic_arrow_back_circle.js

</details>

<details>
<summary>Review comments generated (0)</summary>

* Review: 0
* LGTM: 0

</details>

---

<details>
<summary>Tips</summary>

### Chat with Image description  CodeRabbit Bot (`@coderabbitai`)
- Reply on review comments left by this bot to ask follow-up questions. A review comment is a comment on a diff or a file.
- Invite the bot into a review comment chain by tagging `@coderabbitai` in a reply.

### Code suggestions
- The bot may make code suggestions, but please review them carefully before committing since the line number ranges may be misaligned. 
- You can edit the comment made by the bot and manually tweak the suggestion if it is slightly off.

### Pausing incremental reviews
- Add `@coderabbitai: ignore` anywhere in the PR description to pause further reviews from the bot.

</details>

**Review by bastian-purrer — COMMENTED** [review]
Image description   CodeRabbit

<details>
<summary>Commits</summary>
Files that changed from the base of the PR and between f529e187755df74abacb63d2e650152aacf75a7a and 737493c3a303ae4f590517cd596caeaa8af60d8c commits.
</details>

<details>
<summary>Files selected (1)</summary>

* src/assets/arrow/Ic_arrow_back_circle.js (1)
</details>







<details>
<summary>Files skipped from review due to trivial changes (1)</summary>

* src/assets/arrow/Ic_arrow_back_circle.js

</details>

<details>
<summary>Review comments generated (0)</summary>

* Review: 0
* LGTM: 0

</details>

---

<details>
<summary>Tips</summary>

### Chat with Image description  CodeRabbit Bot (`@coderabbitai`)
- Reply on review comments left by this bot to ask follow-up questions. A review comment is a comment on a diff or a file.
- Invite the bot into a review comment chain by tagging `@coderabbitai` in a reply.

### Code suggestions
- The bot may make code suggestions, but please review them carefully before committing since the line number ranges may be misaligned. 
- You can edit the comment made by the bot and manually tweak the suggestion if it is slightly off.

### Pausing incremental reviews
- Add `@coderabbitai: ignore` anywhere in the PR description to pause further reviews from the bot.

</details>

**Review by bastian-purrer — COMMENTED** [review]
Image description   CodeRabbit

<details>
<summary>Commits</summary>
Files that changed from the base of the PR and between f529e187755df74abacb63d2e650152aacf75a7a and 737493c3a303ae4f590517cd596caeaa8af60d8c commits.
</details>

<details>
<summary>Files selected (1)</summary>

* src/assets/arrow/Ic_arrow_back_circle.js (1)
</details>







<details>
<summary>Files skipped from review due to trivial changes (1)</summary>

* src/assets/arrow/Ic_arrow_back_circle.js

</details>

<details>
<summary>Review comments generated (0)</summary>

* Review: 0
* LGTM: 0

</details>

---

<details>
<summary>Tips</summary>

### Chat with Image description  CodeRabbit Bot (`@coderabbitai`)
- Reply on review comments left by this bot to ask follow-up questions. A review comment is a comment on a diff or a file.
- Invite the bot into a review comment chain by tagging `@coderabbitai` in a reply.

### Code suggestions
- The bot may make code suggestions, but please review them carefully before committing since the line number ranges may be misaligned. 
- You can edit the comment made by the bot and manually tweak the suggestion if it is slightly off.

### Pausing incremental reviews
- Add `@coderabbitai: ignore` anywhere in the PR description to pause further reviews from the bot.

</details>

**Pull request merged**